CMakeLists.txt

```diff
@@ -19,7 +19,6 @@ cmake_minimum_required(VERSION 3.10)
 project(hazelcast-cpp-client
         VERSION 4.0.1
         DESCRIPTION "Hazelcast C++ Client"
-        HOMEPAGE_URL https://hazelcast.org/clients/cplusplus/
```

Comment:
why?

Reply:
I updated the description. See the 2nd item.